src/etl/NFTHelpers.cpp

```diff
@@ -342,7 +346,7 @@ getNFTokenCreateOfferData(
 }
 
 std::pair<std::vector<NFTTransactionsData>, std::optional<NFTsData>>
```

Comment:
Hm, I see now that it was there before. Maybe we can refactor it to use a `struct` or at least use a `typedef`?

Reply:
i think we should tackle this later